internal/cmd/delete/clusterorder/delete_clusterorder_cmd.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright (c) 2025 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the
+License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+language governing permissions and limitations under the License.
+*/
+
+package clusterorder
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	fulfillmentv1 "github.com/innabox/fulfillment-cli/internal/api/fulfillment/v1"
+	"github.com/innabox/fulfillment-cli/internal/config"
+)
+
+func Cmd() *cobra.Command {
+	runner := &runnerContext{}
+	result := &cobra.Command{
+		Use:     "clusterorder [flags] ID",
+		Aliases: []string{"clusterorders"},
+		Short:   "Delete a cluster order",
+		RunE:    runner.run,
+	}
+	return result
+}
+
+type runnerContext struct {
+}
+
+func (c *runnerContext) run(cmd *cobra.Command, args []string) error {
+        // Check that there is exactly one cluster order ID specified
+        if len(args) != 1 {
+	        fmt.Fprintf(
+                        os.Stderr,
+                        "Expected exactly one cluster order ID\n",
+	        )
+                os.Exit(1)
+        }
+	orderId := args[0]
+
+	// Get the context:
+	ctx := cmd.Context()
+
+	// Get the configuration:
+	cfg, err := config.Load()
+	if err != nil {
+		return err
+	}
+	if cfg.Address == "" {
+		return fmt.Errorf("there is no configuration, run the 'login' command")
+	}
+
+	// Create the gRPC connection from the configuration:
+	conn, err := cfg.Connect()
+	if err != nil {
+		return fmt.Errorf("failed to create gRPC connection: %w", err)
+	}
+
+	// Create the client for the cluster orders service:
+	client := fulfillmentv1.NewClusterOrdersClient(conn)
+
+	// Get the order:
+	_, err = client.Get(ctx, &fulfillmentv1.ClusterOrdersGetRequest{
+		Id: orderId,
+	})
+	if err != nil {
+		return fmt.Errorf("failed to retrieve order: %w", err)
+	}
```

Comment:
I'd say that it isn't necessary to retrieve the object first: the `Delete` method below should return an error if the object doesn't exist. If it doesn't then that is a bug in the server.

Reply:
It does return an error, just one that I thought might be a bit deceptive, as it implies that the object may exist:

```
$ ./fulfillment-cli delete clusterorder foo
Error: failed to delete order: rpc error: code = Internal desc = failed to delete object
```

But I can remove this check if you think the server should return a different error if the object doesn't exist?

Reply:
I see, I'd say that is a bug in the server. Lets merge this as is and we change it when the server is fixed.